Services/data/devicemodels/scripts/SetFuelLevel-method.js

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+/*global log*/
+/*global updateState*/
+/*global updateProperty*/
+/*global sleep*/
+/*jslint node: true*/
+
+"use strict";
+
+/**
+ * Entry point function called by the simulation engine.
+ *
+ * @param context        The context contains current time, device model and id
+ * @param previousState  The device state since the last iteration
+ * @param previousProperties  The device properties since the last iteration
+ */
+
+/*jslint unparam: true*/
+function main(context, previousState, previousProperties) {
+
+    // General info indicating that this method is being called
+    log("Executing JavaScript SetFuelLevel method.");
```

Comment:
super-(super?)-nit:  below you put the method in single quotes, 'SetFuelLevel', here you don't.

Reply:
Fixed.